src/components/volunteers/Volunteer/index.tsx

```diff
@@ -58,30 +57,48 @@ export default function Volunteer({
           <Typography variant="h3" pt={2} pb={2}>
             {volunteer.firstName} {volunteer.lastName}
           </Typography>
-          <Box sx={{ display: 'flex' }}>
-            <Typography variant="subtitle1" sx={{ fontWeight: 'bold', pr: 1 }}>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant="h6" sx={{ fontWeight: 'bold', pr: 1 }}>
               Email:
             </Typography>
-            <Typography display="inline">{volunteer.email}</Typography>
+            <Typography display="inline" variant="h6">
+              {volunteer.email}
+            </Typography>
           </Box>
-          <Box sx={{ display: 'flex' }}>
-            <Typography variant="subtitle1" sx={{ fontWeight: 'bold', pr: 1 }}>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant="h6" sx={{ fontWeight: 'bold', pr: 1 }}>
               Phone number:
             </Typography>
-            <Typography display="inline">{formattedPhoneNumber}</Typography>
+            <Typography display="inline" variant="h6">
+              {formattedPhoneNumber}
+            </Typography>
           </Box>
-          <Box sx={{ display: 'flex' }}>
-            <Typography variant="subtitle1" sx={{ fontWeight: 'bold', pr: 1 }}>
-              Previous role:
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant="h6" sx={{ fontWeight: 'bold', pr: 1 }}>
+              Adress:
+            </Typography>
+            <Typography display="inline" variant="h6">
+              {volunteer.address}
             </Typography>
-            <Typography display="inline">{volunteer.previousRole}</Typography>
           </Box>
-          <Box sx={{ display: 'flex' }}>
-            <Typography variant="subtitle1" sx={{ fontWeight: 'bold', pr: 1 }}>
+          <Box sx={{ display: 'flex', alignItems: 'center' }}>
+            <Typography variant="h6" sx={{ fontWeight: 'bold', pr: 1 }}>
               Previous organization:
             </Typography>
-            <Typography display="inline">
-              {volunteer.previousOrganization?.name}
+            <Typography display="inline" variant="h6">
+              <ListItemButton
+                key={volunteer.previousOrganization?._id}
+                onClick={() =>
+                  router.push(
+                    `/organizations/${volunteer.previousOrganization?._id}`
+                  )
+                }
+              >
```

Comment:
If trivial, standardize the gaps between "Address" and "Previous Organization" so it matches the spacing between the rest of the lines. Don't spend a lot of time on it, if it takes more than 5 mins, make a follow-up tech debt story and put it in the backlog to fix it

![image](https://github.com/hack4impact-utk/compassion-ministries/assets/71574118/f3859df7-71fb-4f92-8e53-e087685c0882)